settings.py

```diff
@@ -185,7 +186,10 @@
     'django.core.context_processors.i18n',
 ]
 
-TEMPLATE_DIRS = []
+location = lambda x: os.path.join(os.path.dirname(os.path.realpath(__file__)), x)
```

Comment:
Could also use `FILEPATH`, which is similar to `BASE_DIR` and also defined above.